app/assets/stylesheets/partials/_search.scss

```diff
@@ -71,6 +63,22 @@
         color: $black;
       }
     }
+    #advanced-search-label,
+    #geobox-search-label,
+    #geodistance-search-label {
+      &::before {
+        content: '\f054';
+      }
+    }
+    &[open] {
```

Comment:
Full disclosure: I learned it while writing this refactor. 😅 In fact, I think the reason we'd been using classes instead of the `open` attribute to handle the styles is because I wasn't aware of this convention!